CMSgov/design-system
# [NO-TICKET] gatsby code snippet formatting

## Summary

Implemented prismjs (what we are using on current doc site) to do syntax highlighting in code blocks coming from markdown.

The changes to the `syntax highlighting` styles use Eric's accessible syntax highlighting themes from [this repo](https://github.com/ericwbailey/a11y-syntax-highlighting)

## How to test

`yarn install` & `yarn start:gatsby`

In this example, you can see syntax highlighting for javascript & html

<img width="951" alt="Screen Shot 2022-04-25 at 1 14 38 PM" src="https://user-images.githubusercontent.com/33579665/165158407-33d4088c-feda-448c-8db1-8b70cc41d4f9.png">
